### PR TITLE
update docker container docs

### LINKED ISSUE #1125: Documentation for using docker container out of date for usbipd 4.0.0 (VSC-1283)
### OS

Windows

### Operating System version

Windows 10 WSL2

### Visual Studio Code version

1.86.0

### ESP-IDF version

4.4.3

### Python version

3.8.7

### Doctor command output

N/A

### Extension

_No response_

### Description

Documentation found [here](https://github.com/espressif/vscode-esp-idf-extension/blob/master/docs/tutorial/using-docker-container.md#using-docker-container) is out of date with the most recent version of `usbipd 4.0.0` which has breaking changes. Documentation for these changes can be found [here](https://github.com/dorssel/usbipd-win?tab=readme-ov-file#wsl-2). 

### Debug Message

```plain
n/a
```

### Other Steps to Reproduce

_No response_

### I have checked existing issues, online documentation and the Troubleshooting Guide

- [X] I confirm I have checked existing issues, online documentation and Troubleshooting guide.

### PR DESCRIPTION
**Pull request by brianignacio5**
## Description

Update docker and WSL docs to usbipd latest version changes.

Fixes #1125 

## Type of change

Please delete options that are not relevant.
- This change requires a documentation update

## Steps to test this pull request

Check documentation

## How has this been tested?

Try steps in documentation to set up docker and wsl

**Test Configuration**:
* ESP-IDF Version: 5.3
* OS (Windows,Linux and macOS): Windows

## Checklist
- [x] PR Self Reviewed
- [x] Applied Code formatting
- [x] Added Documentation
- [x] Added Unit Test
- [x] Verified on all platforms - Windows,Linux and macOS

### CODE REVIEW

File: docs/WSL.md [hunk 1]
```diff
@@ -31,22 +24,14 @@ apt-get update
 
 at this moment, local windows and WSL have the `usbipd`. Now check if they are working well on both side, with the following steps:
 
-1. <span id="usbipd_instructions"></span>open PowerShell command prompt with administrator right and then type in the command `usbipd wsl list`:
-
-   <img src="../media\tutorials\using_docker_container\usbipd_wsl_l.png" alt="" height="">
-
-   as you can see, all USB devices from windows have been found and not attached sate.
+1. <span id="usbipd_instructions"></span>open PowerShell command prompt with administrator right and then type in the command `usbipd list` for a list of USB serial devices.
```
Comment (radurentea) [inline docs/WSL.md:27]: Typo: it should be administrator rights

File: docs/tutorial/using-docker-container.md [hunk 2]
```diff
@@ -62,12 +58,12 @@ the default version of WSL is 1 after installing, it needs to upgrade to version
 1. please upgrade to version 2, if not
 
    ```c
-   wsl --set-version Ubuntu-20.04 2
+   wsl --set-version Ubuntu 2
    ```
 
-1. set the distribution, as below:
+1. set the distribution as default, as below:
```
Comment (radurentea) [inline docs/tutorial/using-docker-container.md:64]: I think it would sound better if we avoid the double "as". Maybe something like:
"configure the distribution to be the default option, as specified below:"

File: docs/tutorial/using-docker-container.md [hunk 3]
```diff
@@ -76,19 +72,12 @@ at last, to check if the commands have taken effect with `wsl --status` command.
 
 ## Docker Desktop for Windows
 
-As the distribution Ubuntu 20.04 has been updated to version 2, so it needs to modify accordingly from docker side and choose the Ubuntu 20.04 as the default WSL integration as well.
+As the distribution Ubuntu has been updated to version 2, so it needs to modify accordingly from docker side and choose the Ubuntu as the default WSL integration as well.
```
Comment (radurentea) [inline docs/tutorial/using-docker-container.md:75]: I would rephrase this to:

As the Ubuntu distribution has been updated to version 2, adjustments on the Docker side are required, and Ubuntu should also be chosen as the default WSL integration.

File: docs/tutorial/using-docker-container.md [hunk 4]
```diff
@@ -99,21 +88,13 @@ apt-get update
...
-1. <span id="usbipd_instructions"></span>open PowerShell command prompt with administrator right and then type in the command `usbipd wsl list`:
-
-   <img src="../../media\tutorials\using_docker_container\usbipd_wsl_l.png" alt="" height="">
-
-   as you can see, all USB devices from Windows have been found and not attached sate.
-
-2. to access the specified device from local Windows on WSL, it needs to bind this device. Open PowerShell command prompt with administrator rights and then type in the command `usbipd bind -b <BUSID>`:
-
-   <img src="../../media\tutorials\using_docker_container\usbipd_bind.png" alt="" height="">
+1. <span id="usbipd_instructions"></span>open PowerShell command prompt with administrator right and then type in the command `usbipd list` for a list of USB serial devices.
```
Comment (radurentea) [inline docs/tutorial/using-docker-container.md:91]: administrator rights

File: docs/tutorial/using-docker-container.md [hunk 5]
```diff
@@ -99,21 +88,13 @@ apt-get update
...
 
-   **Note**: this command needs to be used only one time,unless the computer has restarted. **1-1** is the device's bus id I would like to bind.
+2. to access the specified device from local Windows on WSL, it needs to bind this device. Open PowerShell command prompt with administrator rights and then type in the command `usbipd bind --busid <BUSID>`:
```
Comment (radurentea) [inline docs/tutorial/using-docker-container.md:93]: I would rephrase the first part to:

to access the specified device from Windows on WSL locally, the device must be bound.

File: docs/tutorial/using-docker-container.md [hunk 6]
```diff
@@ -99,21 +88,13 @@ apt-get update
...
 
-3. after binding, please attach the specified device to WSL with `usbipd wsl attach --busid 1-1` command in the powershell command prompt.
+   **Note**: this command needs to be used only one time,unless the computer has restarted. **1-1** is the device's bus id `<BUSID>` I would like to bind.
 
-<img src="../../media\tutorials\using_docker_container\usbipd_wsl_attach.png" alt="" height="">
+3. after binding, please attach the specified device to WSL with `usbipd attach --wsl --busid <BUSID>` command in the powershell command prompt.
```
Comment (radurentea) [inline docs/tutorial/using-docker-container.md:97]: We should keep consistency here, so we should keep using "PowerShell" instead of "powershell"

File: docs/tutorial/wsl.md [hunk 7]
```diff
@@ -90,21 +81,13 @@ apt-get update
...
-1. <span id="usbipd_instructions"></span>open PowerShell command prompt with administrator right and then type in the command `usbipd wsl list`:
-
-   <img src="../../media\tutorials\using_docker_container\usbipd_wsl_l.png" alt="" height="">
-
-   as you can see, all USB devices from Windows have been found and not attached sate.
-
-2. to access the specified device from local Windows on WSL, it needs to bind this device. Open PowerShell command prompt with administrator rights and then type in the command `usbipd bind -b <BUSID>`:
-
-   <img src="../../media\tutorials\using_docker_container\usbipd_bind.png" alt="" height="">
+1. <span id="usbipd_instructions"></span>open PowerShell command prompt with administrator right and then type in the command `usbipd list` for a list of USB serial devices.
```
Comment (radurentea) [inline docs/tutorial/wsl.md:84]: rights

File: docs/tutorial/wsl.md [hunk 8]
```diff
@@ -90,21 +81,13 @@ apt-get update
...
 
-   **Note**: this command needs to be used only one time,unless the computer has restarted. **1-1** is the device's bus id I would like to bind.
+2. to access the specified device from local Windows on WSL, it needs to bind this device. Open PowerShell command prompt with administrator rights and then type in the command `usbipd bind --busid <BUSID>`:
```
Comment (radurentea) [inline docs/tutorial/wsl.md:86]: same as above:

to access the specified device from Windows on WSL locally, the device must be bound.

**Review by radurentea — COMMENTED** [review]
I've gone through the changes and left some comments addressing a few typos that need correction. Additionally, I've suggested some rephrasing in certain parts to enhance clarity and readability.

File: docs/tutorial/using-docker-container.md [hunk 9]
```diff
@@ -88,13 +88,13 @@ apt-get update
...
+1. <span id="usbipd_instructions"></span>open PowerShell command prompt with administrator rights and then type in the command `usbipd list` for a list of USB serial devices.
 
-2. to access the specified device from local Windows on WSL, it needs to bind this device. Open PowerShell command prompt with administrator rights and then type in the command `usbipd bind --busid <BUSID>`:
+2. To access the specified device from Windows on WSL locally, the device must be bound with **usbipd**. Open PowerShell command prompt with administrator rights and then type in the command `usbipd bind --busid <BUSID>`:
 
    **Note**: this command needs to be used only one time,unless the computer has restarted. **1-1** is the device's bus id `<BUSID>` I would like to bind.
 
-3. after binding, please attach the specified device to WSL with `usbipd attach --wsl --busid <BUSID>` command in the powershell command prompt.
+3. after binding, please attach the specified device to WSL with `usbipd attach --wsl --busid <BUSID>` command in the Powershell command prompt.
```
Comment (radurentea) [inline docs/tutorial/using-docker-container.md:97]: Small typo here, sorry! 😄   In the rest of this documentation we've used PowerShell, so I think we should stick with it for consistency sake.

**Review by radurentea — COMMENTED** [review]
There is one more typo

**Comment by AndriiFilippov** [comment]
@brianignacio5 hi ! 

image

**Comment by AndriiFilippov** [comment]
@brianignacio5 
Perhaps it is necessary to increase the font and highlight this text because these steps are practically imperceptible. 
In my opinion 
what do you think ?

**Comment by AndriiFilippov** [comment]
@brianignacio5 hi !

LGTM 👍

**Review by AndriiFilippov — APPROVED** [review]
